apps/web/src/services/report.ts

```diff
@@ -219,9 +223,23 @@ const reportCreators: ReportCreator[] = [
     name: 'Report 8',
     description: 'Number of Internationally Educated Nurse Registrants Working in BC',
     header: (data: Record<string, string | number>[]): string[] => {
-      return ['', ...Object.keys(data[0]).slice(1)];
+      return [
+        '',
+        ...Object.keys(data[0])
+          .slice(1)
+          .map(v => {
+            return v.includes('current_period') ? `${v.split(' ')[0]}*` : v;
+          }),
+      ];
+    },
+    colWidths: [35, 20, 20, 20],
+    notes: (data: Record<string, string | number>[]): any[] => {
```

Comment:
Notes is a bit vague for a variable name. It would be good to add a small dock block to explain the intent behind it.

Reply:
I'll add some comments for clarification